server/prisma/schema.prisma

```diff
@@ -200,13 +200,30 @@ model Group {
   memberActivity MemberActivity[]
   patronages     Patron[]
   socialLinks    GroupSocialLinks[]
+  roleOrder      GroupRoleOrder[]
```

Comment:
technically should be `roleOrders` since it's an array

Reply:
done